compute.js

```diff
@@ -94,7 +94,9 @@ class ComputePipeline
         {
             this.destroyResources();
             let usage = GPUBufferUsage.STORAGE | GPUBufferUsage.COPY_SRC;
-            const numberElements = windowSize[0] * windowSize[1];
+
+            // we will fix the size of the workgroup to be 32x32 for printMain.
```

Comment:
just use (1,1,1) for printmain.

Reply:
we just want to use 1 thread?

Reply:
I made it to (2, 2, 1), because I want `dispatchThreadID` still to be useful.